src/components/DevTools/DevTools.wallet.tsx

```diff
@@ -44,56 +44,52 @@ class BsIdentity {
     identity: Identity,
     phrase: string,
     alias: string
-  ): Promise<void | IMyIdentity> {
+  ): void | IMyIdentity {
     const selectedIdentity:
       | IMyIdentity
       | undefined = Wallet.getSelectedIdentity(
       persistentStoreInstance.store.getState()
     )
 
-    return new Promise((resolve, reject) => {
-      if (!selectedIdentity) {
-        return reject(new Error('No Identity selected'))
-      }
-      return BalanceUtilities.makeTransfer(
-        selectedIdentity,
-        identity.address,
-        ENDOWMENT,
-        () => {
-          const newContact: IContact = {
-            metaData: {
-              name: alias,
-            },
-            publicIdentity: identity.getPublicIdentity(),
-          }
-          persistentStoreInstance.store.dispatch(
-            Contacts.Store.addContact(newContact)
-          )
+    if (!selectedIdentity) {
+      throw new Error('No Identity selected')
+    }
+    return BalanceUtilities.makeTransfer(
```

Comment:
That function seems wrong. It was resolving the Promise with the `newIdentity` in the callback. Now it is returning whatever `BalanceUtilities.makeTransfer` returns

Reply:
I should just revert it, then return a `Promise<void | IMyIdentity>` where it was used.